.gitignore

```diff
@@ -67,3 +66,4 @@ site_scons/site_tools/mfprogram/*.pyc
 site_scons/site_tools/gch/*.pyc
 
 screenshot_*
+TODO.txt
```

Comment:
Not sure what this is for

Reply:
Oh i wrote a file that i used to keep track of what I was doing, didn't realize that would be included in the PR, will remove this. got exams coming up so will probably wont get back to this for a few weeks.